core/primitives/digest.hpp

```diff
@@ -175,18 +175,22 @@ namespace kagome::primitives {
     return s >> dic.consensus_engine_id >> dic.data;
   }
 
+  /// Runtime code or heap pages updated.
+  struct RuntimeEnvironmentUpdated : public Unused<8> {};
```

Comment:
In substrate it is zero-sized type, and any parent class is needless here.
```suggestion
  struct RuntimeEnvironmentUpdated {};
```
And template-arg of **Unused** serves only for differ on other ones.